TypeScript errors in app build

Following errors when running `npm run build`:

```
#10 7.556 src/services/apimService.ts(133,34): error TS2339: Property 'id' does not exist on type 'unknown'.
#10 7.557 src/services/monetizationService.ts(14,54): error TS2339: Property 'find' does not exist on type 'unknown'.
```

--- a/app/src/services/apimService.ts
+++ b/app/src/services/apimService.ts
@@ -125,7 +125,7 @@ export class ApimService {
             const url = `${managementApiUrl}/identity?api-version=2019-12-01`
             const response = await fetch(url, { method: "GET", headers: { Authorization: credentials } });
             const sasToken = response.headers.get("Ocp-Apim-Sas-Token");
-            const identity = await response.json();
+            const identity = await response.json() as { id: string };
 
             return {
                 authenticated: true,

--- a/app/src/services/monetizationService.ts
+++ b/app/src/services/monetizationService.ts
@@ -1,15 +1,22 @@
 import { ProductContract } from "@azure/arm-apimanagement/esm/models";
 import fetch from "node-fetch";
 
+export interface MonetizationModel {
+    id: string,
+    prices: any;
+    pricingModelType: string
+    recurring: any;
+}
+
 export class MonetizationService {
     /** Retrieve our defined monetization model for a given APIM product */
-    public static async getMonetizationModelFromProduct(product: ProductContract) {
+    public static async getMonetizationModelFromProduct(product: ProductContract): Promise<MonetizationModel> {
 
         const monetizationModelsUrl = `${process.env.APIM_GATEWAY_URL}/billing/monetizationModels`;
 
         const monetizationModelsResponse = await fetch(monetizationModelsUrl, { method: "GET", headers: { "Ocp-Apim-Subscription-Key": process.env.APIM_ADMIN_SUBSCRIPTION_KEY } });
 
-        const monetizationModels = (await monetizationModelsResponse.json());
+        const monetizationModels = (await monetizationModelsResponse.json()) as MonetizationModel[];
 
         const monetizationModel = monetizationModels.find((x: { id: any; }) => x.id === product.name);
 

